nuitka/build/static_src/MainProgram.c

```diff
@@ -1191,6 +1191,9 @@ int __stdcall wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, wchar_t *lp
 #else
 #if defined(_WIN32)
 int wmain(int argc, wchar_t **argv) {
+#if defined(_NUITKA_HIDE_CONSOLE_WINDOW)
```

Comment:
The placement here is probably very agressive, but also why not, need to check though.

Reply:
Probably might make sense to merge this with

```
    // Attach to the parent console respecting redirection only, otherwise we
    // cannot even output traces.
#if defined(_WIN32) && defined(_NUITKA_ATTACH_CONSOLE_WINDOW)
    inheritAttachedConsole();
#endif
```

kind of code and just have a `setupIneritedConsole()` helper that does these condition checks in the common code rather than per item.

Reply:
I will probably clean this up at some point, but not for release, not really that much of an improvement right now.